core/includes/theme.inc

```diff
@@ -1927,12 +1927,35 @@ function theme_dropbutton_wrapper($variables) {
 }
 
 /**
+ * Preprocess variables for theme_image().
+ *
+ * @since 1.8.0 Function added. Populates $variables['attributes'] before
+ *   theme_image() is called.
+ */
+function template_preprocess_image(&$variables) {
+  // Accept either URI or path key. But both are handled by file_create_url().
+  $path_or_uri = $variables['uri'] ? $variables['uri'] : $variables['path'];
```

Comment:
Both `uri` and `path` will exist in the `$variables` array thanks to `backdrop_common_theme()` where `theme_image` is registered. Both will have a value of `NULL` though. Ultimately this just means that if neither are passed in, you'll end up with `<img src="" />`, which probably won't be what you want but would be the same result as missing any other theme parameters throughout Backdrop.